src/object/player.cpp

```diff
@@ -254,6 +254,12 @@ Player::Player(PlayerStatus& player_status, const std::string& name_, int player
   SoundManager::current()->preload("sounds/invincible_start.ogg");
   SoundManager::current()->preload("sounds/splash.wav");
   SoundManager::current()->preload("sounds/grow.wav");
+  m_bubble_particles[0] = Surface::from_file("images/particles/air_bubble-1.png");
+  m_bubble_particles[1] = Surface::from_file("images/particles/air_bubble-2.png");
+  m_bubble_particles[2] = Surface::from_file("images/particles/air_bubble-3.png");
+  m_bubble_particles[3] = Surface::from_file("images/particles/air_bubble-4.png");
+  m_bubble_timer.start(3.0f + (rand() % 2));
```

Comment:
For usage of rng in graphics effects, use `graphicsRandom`
```suggestion
  m_bubble_timer.start(3.0f + graphicsRandom.randf(2));
```